### PR TITLE
Clarifiy unexpected Packet private CIDR behaviour

### PR DESCRIPTION
**Pull request by DGollings**
Attempts to address or at least mitigate issue #159

### CODE REVIEW

File: docs/flatcar-linux/packet.md [hunk 1]
```diff
@@ -110,6 +110,10 @@ module "controller" {
   # Check yours from the `IPs & Networks` tab from your Packet.net account.
   # If an IP block is not allocated yet, try provisioning an instance from the console in
   # that region. Packet will allocate a public IP CIDR.
+  # Note: the private CIDR range has been observed to change unexpectedly, 
+  # although at the time it was an empty/dormant Packet project and there had been some networking upgrades
+  # be sure to manually check this value again when either reusing an older project 
+  # or see unexplainable networking behaviour (inter-node timeouts)
```
Comment (rata) [inline docs/flatcar-linux/packet.md]: What about something more clear (doormant Packet project is not a term I'm familiar with)? Like this:

```suggestion
# Note: Packet does not guarantee this CIDR to be stable if there are no servers deployed in the project and region
```

**Review by rata — COMMENTED** [review]
Can you please update the PR title to be more representative of what the change does? :)

File: docs/flatcar-linux/packet.md (hunk 1 above)
Comment (DGollings) [reply docs/flatcar-linux/packet.md]: dormant means asleep, unused. But sure, that's also fine/clearer :)

File: docs/flatcar-linux/packet.md [hunk 2]
```diff
@@ -110,6 +110,10 @@ module "controller" {
   # Check yours from the `IPs & Networks` tab from your Packet.net account.
   # If an IP block is not allocated yet, try provisioning an instance from the console in
   # that region. Packet will allocate a public IP CIDR.
+  # Note: the private CIDR range has been observed to change unexpectedly, 
+  # although at the time it was an empty/dormant Packet project and there had been some networking upgrades
+  # be sure to manually check this value again when either reusing an older project 
+# Note: Packet does not guarantee this CIDR to be stable if there are no servers deployed in the project and region
```
Comment (rata) [inline docs/flatcar-linux/packet.md]: Sorry I wasn't clear, this is just duplicating the notes and seems more confusing, IMHO. Just the second one seems fine. What do you think?

(also, spacing is wrong)

**Review by rata — COMMENTED** [review]
Also, please update the commit titles to reflect the changes, (not just the unsuful gihub default of "Update <file>", squash them if possible :)

File: docs/flatcar-linux/packet.md (hunk 2 above)
Comment (DGollings) [reply docs/flatcar-linux/packet.md]: sure, this line is fine

**Comment by DGollings** [comment]
> Also, please update the commit titles to reflect the changes, (not just the unsuful gihub default of "Update ", squash them if possible :)

I'm more of a 'never rewrite (git) history' type of guy, so never bothered to learn how to either rename a commit or squash it, could you point me to the FM and I'll figure it out :)

**Comment by rata** [comment]
@DGollings no worries, just add a different commit then and I can squash on merge :)

**Comment by DGollings** [comment]
Quite the discussion: https://softwareengineering.stackexchange.com/questions/263164/why-squash-git-commits-for-pull-requests

I can see the pro/cons, but trunk based development ftw (?)

done, ish, removed what I think was a duplicate Note:
(edit: although I agree my original commit message was useless)

**Review by rata — APPROVED** [review]
LGTM, thanks again! :-)

**Pull request merged**